lib/tic_tac_toe_checker.ex

```diff
@@ -24,6 +43,7 @@ defmodule TicTacToeChecker do
   end
 
   @impl GenServer
+  @spec handle_cast({:launch, coordinates()}, %{board: board()}) :: {:noreply, atom() | map()}
```

Comment:
I’d better go with 1 spec per function, not per clause.
```suggestion
  @spec handle_cast({:launch, coordinates()} | {:done, [coordinates()]}, %{board: board()}) :: {:noreply, atom() | map()}
```